superset-frontend/spec/javascripts/explore/components/AdhocFilterEditPopover_spec.jsx

```diff
@@ -97,6 +106,14 @@ describe('AdhocFilterEditPopover', () => {
     expect(wrapper.find(AdhocFilterEditPopoverSqlTabContent)).toExist();
   });
 
+  it('renders simple and sql tabs with ErrorBoundary instead of content', () => {
+    const { wrapper } = setup({ adhocFilter: faultiAdhocFilter });
+    expect(wrapper.find(Tabs)).toExist();
+    expect(wrapper.find(Tabs.TabPane)).toHaveLength(2);
+    expect(wrapper.find(Button)).toHaveLength(2);
+    expect(wrapper.find(ErrorBoundary)).toHaveLength(2);
+  });
```

Comment:
```suggestion
  it('renders simple and sql tabs with ErrorBoundary instead of content', () => {
    const { wrapper } = setup({ adhocFilter: faultyAdhocFilter });
    expect(wrapper.find(Tabs)).toExist();
    expect(wrapper.find(Tabs.TabPane)).toHaveLength(2);
    expect(wrapper.find(Button)).toHaveLength(2);
    expect(wrapper.find(ErrorBoundary)).toHaveLength(2);
  });
```